provider/lxd/environ_raw.go

```diff
@@ -112,7 +107,7 @@ func newClient(
 // getRemoteConfig returns a lxdclient.Config using a TCP-based remote
 // if called from within an instance started by the LXD provider. Otherwise,
 // it returns an errors satisfying errors.IsNotFound.
-func getRemoteConfig(readFile readFileFunc, runCommand runCommandFunc) (*lxdclient.Config, error) {
+func getRemoteConfig(readFile readFileFunc, runCommand runCommandFunc, spec environs.CloudSpec) (*lxdclient.Config, error) {
```

Comment:
This breaks some tests, which are calling getRemoteConfig directly. Can you please fix them?